container/src/LuigiCompoundContainer.svelte

```diff
@@ -108,7 +108,17 @@
       return;
     }
     thisComponent.updateContext = (contextObj: any, internal?: any) => {
-      (thisComponent.getNoShadow() ? thisComponent : mainComponent)._luigi_mfe_webcomponent.context = contextObj;
+      if ((thisComponent.getNoShadow() ? thisComponent : mainComponent)._luigi_mfe_webcomponent?.nodeName === 'DIV') {
+        // handle multiple webcomponents on page
+        const wrappers = (thisComponent.getNoShadow() ? thisComponent : mainComponent)._luigi_mfe_webcomponent.querySelectorAll('div[style^="grid-row"]');
+        wrappers?.forEach((item) => {
+          if (item._luigi_mfe_webcomponent) {
+            item._luigi_mfe_webcomponent.context = contextObj;
+          }
+        });
+      } else {
+        (thisComponent.getNoShadow() ? thisComponent : mainComponent)._luigi_mfe_webcomponent.context = contextObj;
+      }
```

Comment:
was there a bug with updateContext not working on Compound Container? If it is, could you create a bug issue and handle it separately there. Then this Cypress Pr can be merged/reviewied easier

Reply:
In this case '_luigi_mfe_webcomponent' points to direct child of MAIN element - which is DIV element wrapping compound item containers created through 'createCompoundItemContainer' method. Each of these containers have some WC for which context should be updated - instead of adding 'context' prop to the DIV wrapper. Check attached screenshot for reference

![Zrzut ekranu 2024-07-02 105131](https://github.com/SAP/luigi/assets/169130331/7806c00a-bd45-4246-a33d-77bbf3124074)